packages/core/src/database/postgres/service.ts

```diff
@@ -307,6 +307,29 @@ export class PostgresDatabaseService implements BaseDatabaseService {
               )} ago`,
             });
 
+            // Remove any indexes, will be recreated once the app
+            // becomes healthy.
+            await Promise.all(
```

Comment:
A nested for loop might be more readable here (in a transaction so no concurrency benefit).